Dockerfile

```diff
@@ -89,10 +89,25 @@ RUN set -x && apt-get update \
   && cd /opt/intel/crypto-api-toolkit \
   # disable building tests
   && sed -i -e 's;test;;g' ./src/Makefile.am \
+  # disable enclave signing inside CTK
+  && sed -i -e '/libp11SgxEnclave.signed.so/d' ./src/p11/trusted/Makefile.am \
```

Comment:
It's odd that they don't have a switch for doing this... I mean patching their Makefile.am with sed works but will be difficult to maintain going forward. No need for you to change anything but I'll go file an issue in crypto-api-toolkit.

Reply:
Yes, I agree patching source brings its own maintenance burden. It's the quickest solution at the moment ;-).